auth/auth-store/src/main/java/com/bazaarvoice/emodb/auth/permissions/TablePermissionManager.java

```diff
@@ -73,34 +69,53 @@ public TablePermissionManager(PermissionResolver permissionResolver, DataStore d
     }
 
     @Override
-    public void updateForRole(String role, PermissionUpdateRequest request) {
-        update(getRoleKey(role), request);
-    }
-
-    private void update(String key, PermissionUpdateRequest request) {
+    public void updatePermissions(String id, PermissionUpdateRequest request) {
+        checkNotNull(id, "id");
         checkNotNull(request, "request");
         validateTable();
 
```

Comment:
`isRevokeRest()` is a primitive boolean so it can't be null and either `true` or `false` is always valid, so unless there's something deeper I'm missing there's nothing that needs pre-validation here.